.projenrc.js

```diff
@@ -28,6 +28,44 @@ const project = new web.ReactTypeScriptProject({
   devDeps: ["@types/react-router-dom"],
 });
 
+function addStorybook() {
+  project.addDevDeps(
+    "@storybook/addon-actions",
+    "@storybook/addon-essentials",
+    "@storybook/addon-links",
+    "@storybook/node-logger",
+    "@storybook/preset-create-react-app",
+    "@storybook/react",
+    "babel-loader@8.1.0"
+  );
+
+  // Add tasks and config for storybook
+  project.addTask("storybook", {
+    exec: "start-storybook -p 6006 -s public",
+    description: "run local storybook server",
+  });
+
+  project.addTask("build:storybook", {
```

Comment:
Should this part of the main package compile?

```ts
const build = project.addTask(..);
project.compileTask.spawn(build);
```

I'd also call this `storybook:build` instead of `build:storybook`